frontend/test/metabase-lib/lib/Dimension.unit.spec.js

```diff
@@ -138,26 +138,6 @@ describe("Dimension", () => {
   });
 
   describe("INSTANCE METHODS", () => {
-    describe("dimensions()", () => {
-      it("returns `dimension_options` of the underlying field if available", () => {
-        pending();
-      });
-      it("returns sub-dimensions for matching dimension if no `dimension_options`", () => {
-        // just a single scenario should be sufficient here as we will test
-        // `static dimensions()` individually for each dimension
-        pending();
-      });
```

Comment:
Optionally, we can use [`test.todo("...")`](https://jestjs.io/docs/api#testtodoname) if we want to keep the tests

Reply:
Yeah, I lean toward blindly removing code that has been dead for 3+ years. I'm not sure how many of these test descriptions are outdates/no longer relevant. Whoever ends up writing all the unit tests for this code (hopefully soon 🙂 ) should probably approach it with fresh eyes.